### PR TITLE
first commit - added theme working

### LINKED ISSUE #922: [Docs] Update contributor guide with enhanced with how dark mode works
**Current State:**
Meshery.io has a new dark mode.

**Desired State:**
New and existing contributors need training / documentation on how this feature works. 

#### Acceptance Criteria
1. Concise instructions and example(s) should be included in this repo's readme.
2. A similar set of instructions should be included in a new page in https://docs.meshery.io/project/contributing

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io)
- [Meshery documentation source](https://github.com/meshery/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Shivam-AfA**
Signed-off-by: Shivam-AfA <88040318+Shivam-AfA@users.noreply.github.com>

**Description**

This PR fixes #922 

**Notes for Reviewers**
This PR adds the documentation for new contributers to understand how they can contribute to Meshery.io website based on light and dark theme features on the website.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Shivam-AfA** [comment]
@leecalcote Kindly review. And I have a suggestion, we can also add the local environment setup guide in the CONTRIBUTING.md file. It is currently added to the README.md file.

File: integrations/data.json [hunk 1]
```diff
@@ -1 +1 @@
...r":"../images/integration/kubevirt-color.svg","white":"../images/integration/kubevirt-white.svg"},{"name":"Athenz","color":"../images/integration/athenz-color.svg","white":"../images/integration/athenz-white.svg"},{"name":"Flagger","color":"../images/integration/flagger-color.svg","white":"../images/integration/flagger-white.svg"},{"name":"Piraeus","color":"../images/integration/piraeus datastore-color.svg","white":"../images/integration/piraeus datastore-white.svg"},{"name":"Citrix Service Mesh","color":"../images/integration/cpx-color.svg","white":"../images/integration/cpx-white.svg"},{"name":"Kyverno","color":"../images/integration/kyverno-monitor-color.svg","white":"../images/integration/kyverno-monitor-white.svg"},{"name":"Kubedl","color":"../images/integration/kubedl-color.svg","white":"../images/integration/kubedl-white.svg"},{"name":"Spire","color":"../images/integration/spire-color.svg","white":"../images/integration/spire-white.svg"},{"name":"Kured","color":"../images/integration/kured-color.svg","white":"../images/integration/kured-white.svg"},{"name":"Tinkerbell","color":"../images/integration/tinkerbell-color.svg","white":"../images/integration/tinkerbell-white.svg"}]
```
Comment (Shivam-AfA) [inline integrations/data.json:1]: Kindly ignore this file.

**Comment by Shivam-AfA** [comment]
I also commented out a console log in the code.

**Comment by leecalcote** [comment]
https://github.com/meshery/meshery.io/pull/952/checks?check_run_id=10623970253

**Comment by Shivam-AfA** [comment]
There was an issue with Kuma.svg image so had to create a new PR. Here's the link: https://github.com/meshery/meshery.io/pull/968

**Pull request closed without merging**